contribute/style-guides/backend.md

```diff
@@ -75,3 +75,15 @@ Valid reasons to use a pointer include (but not necessarily limited to):
   allocating heap memory)
 * You might *need* `nil` to tell if a variable isn't set, although usually it's better to use the type's zero
   value to tell instead
+
+## Database
+
```

Comment:
"Avoid stacked headings, which is following one heading with another heading." See https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-style-guide.md#heading-donts